packages/server/src/lib/email.js

```diff
@@ -48,6 +49,9 @@ async function deliverEmail({
 }) {
     let userTags = [];
     const recipientId = await db.getUserIdForEmail(toAddress);
+    const activeContext = tracer.scope().active()?.context();
+    const traceId = activeContext?.toTraceId();
+    const spanId = activeContext?.toSpanId();
```

Comment:
I think we should conditionally include the `dd_trace_id` and `dd_span_id` email tags so that they are only present when not `undefined`.
```suggestion
    const activeContext = tracer.scope().active()?.context();
    const traceId = activeContext?.toTraceId();
    const spanId = activeContext?.toSpanId();
    const ddTraceTags = [];
    if (traceId) {
        ddTraceTags.push(`dd_trace_id=${traceId}`);
        if (spanId) {
            ddTraceTags.push(`dd_span_id=${spanId}`);
        }
    }
```